changelogs/fragments/5145-wdc-redfish-enclosure-power-state

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Please add a `.yml` extension to this file, then all tests should pass :)

Reply:
Turns out the GitHub UI allows you to rename files. I tried it out, and it seems to work, so no more change necessary ;)

Reply:
Sorry about that!